src/bundle/Templating/Twig/SecurityExtension.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\AdminUi\Templating\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+final class SecurityExtension extends AbstractExtension
```

Comment:
@konradoboza Could you please provide integration test for this class ?

Reply:
Provided via https://github.com/ibexa/admin-ui/pull/1556/commits/ef3659b740f2b961fe4305054f43272382e6c253.